packages/flare/bin/cron_job_ingest_events.py

```diff
@@ -151,8 +151,8 @@ def create_collection(app: client.Application) -> None:
         )
 
 
-def save_start_date(app: client.Application, tenant_id: int) -> None:
-    current_tenant_id = get_current_tenant_id(app=app)
+def save_last_ingested_tenant_id(app: client.Application, tenant_id: int) -> None:
```

Comment:
This is tricky because this function does two things:
1. saves the started date
2. saves the last_ingested_tenant_id

So how to best name it? 🤔

Reply:
The first save date was useless since on the first run, the last ingested tenant id is None. Any idea how it should be called?

Reply:
Aaaah I see, said the blind man when he grabbed his hammer and saw...

Reply:
Yeah, dunno about name.. Ugh. naming is worst. This is fine until someone else raises it. 😂